lib/rabblebot/plugins/weather.rb

```diff
@@ -0,0 +1,86 @@
+require 'open-uri'
+require 'json'
+
+class RabbleBot
+  attr_reader :google_api, :forecastio_api
+  
+  module RabbleBotPlugin
+    # a weather plugin
+    class Weather < BasicPlugin
+      def initialize(bot)
```

Comment:
you're going to need to change this to `initialize(bot, config)` due to the changes I just merged